angular.json

```diff
@@ -123,18 +110,6 @@
               "projects/shepherd-tester/src/assets"
             ]
           }
-        },
-        "lint": {
```

Comment:
Why are we removing all the linting?

Reply:
this related to this issue https://github.com/angular-eslint/angular-eslint/issues/715